src/app/zones/views/ZoneListView.vue

```diff
@@ -252,7 +252,13 @@ function transformToTableData(zoneOverviews: ZoneOverview[]): ZoneOverviewTableR
       }
     })
 
-    const status = getItemStatusFromInsight(zoneOverview.zoneInsight)
+    let status: StatusKeyword | 'disabled'
+
+    if (zoneOverview.zone.enabled === false) {
+      status = 'disabled'
+    } else {
+      status = getItemStatusFromInsight(zoneOverview.zoneInsight)
+    }
```

Comment:
same as above

Reply:
Addressed in https://github.com/kumahq/kuma-gui/pull/1297#discussion_r1289659869.